packages/mendel-development/require-transform.js

```diff
@@ -14,15 +14,24 @@ function _wrap(src) {
     return wrapper[0] + src + wrapper[1];
 }
 
-function replaceRequiresOnSource (src, wrap) {
+function replaceRequiresOnSource (src, dirs, wrap) {
     var opts = {
         ecmaVersion: 6,
         allowReturnOutsideFunction: true
     };
     var _src = falafel(src, opts, function (node) {
         if (isRequire(node)) {
             var module = node.arguments[0].value;
-            node.update("__mendel_require__('" + module + "')");
+
+            var match;
+            dirs.some(function(dir) {
+                match = module.split('/' + dir + '/')[1];
+                return !!(match);
+            });
```

Comment:
Again similar to `variationMatches`. Did you / could you double check all those cases and see if they can share helper functions?